### PR TITLE
Create Cornmeal_Molasses_Scones.md

### PR DESCRIPTION
**Pull request by Deadlysmurf**

### CODE REVIEW

File: baking/Cornmeal_Molasses_Scones.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+name: "Cornmeal Molasses Scones"
+source_url: "https://www.seductioninthekitchen.com/cornmeal-molasses-scones/"
+oven_temp: "370 F"
+time: "60m"
+yield: "9 inch pie pan"
+---
+
+# Ingredients
+
```
Comment (hockeybuggy) [inline baking/Cornmeal_Molasses_Scones.md:10]: Would you mind tighting up this whitespace? I prefer one space after a title and two spaces before a title. I should set up `mdlint` (or some equivalent in this repo)

File: baking/Cornmeal_Molasses_Scones.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+- 1/2 cup cornmeal
+- 1/4 cup molasses
+- 3 tablespoons butter
+- 1 teaspoon salt
+- 1 teaspoon cinnamon
+- 1 egg
+- 1 ounce package active dry yeast 1/4
+- 3 cups all-purpose flour
+
+
```
Comment (hockeybuggy) [inline baking/Cornmeal_Molasses_Scones.md:22]: As per comment above could you ditch a line here?

File: baking/Cornmeal_Molasses_Scones.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+1. Combine the water, molasses, and butter. Stir it till it looks like the molasses is dissolved. 
+
+2. Add the yeast, and let the yeast bloom for about 5 minutes. Then add the egg, and stir everything one more time.
+
+3. Next add all the dry ingredients. Combine it until you have a dough ball, cover and let it rise for about 30 minutes.
+
+4. Grease a 9 inch pie plate and spread the dough into the pie plate.
+
+5. Preheat the oven to 375 (letting the dough rise for a secondish time while preheating) and bake the scones for about 25 minutes. You know it is done when sounds hollow when you tap them.
```
Comment (hockeybuggy) [inline baking/Cornmeal_Molasses_Scones.md:34]: Thoughts on moving the preheat step to before step 4? Or is the "secondish time" important?

**Review by hockeybuggy — CHANGES_REQUESTED** [review]
Sorry for the late review.

File: baking/Cornmeal_Molasses_Scones.md (hunk 3 above)
Comment (Deadlysmurf) [reply baking/Cornmeal_Molasses_Scones.md:34]: The second rise time is important to the process, or you end up with a flat puck of sadness and not fluffy scones.

Comment (hockeybuggy) [reply baking/Cornmeal_Molasses_Scones.md:34]: Sounds good.

**Comment by Deadlysmurf** [comment]
I seem to have misplaced the branch for this so don't seem to be able to edit - thoughts?

**Comment by hockeybuggy** [comment]
Huh. Ya. Looks like the fork for this is gone. You could recreate this PR if you copy the raw: https://raw.githubusercontent.com/hockeybuggy/recipes/c435f3c95201619a6aa50b72fd8c9406ae996e99/baking/Cornmeal_Molasses_Scones.md

But I think this PR would have to be closed and a new one opened up..

**Comment by hockeybuggy** [comment]
:thumbsup:

**Pull request closed without merging**